src/utils/one-time-code-utils.js

```diff
@@ -0,0 +1,18 @@
+export function saveCodeInLocalStorage(
+  connectedAccount,
+  disputeId,
+  oneTimeCode
+) {
+  localStorage.setItem(
+    `oneTimeCode-${connectedAccount}-${disputeId}`,
```

Comment:
In the client we use `:` as delimiters for some localstorage keys (no real reason, but I guess it's more rare for a variable to include `:` than a `-`).

Reply:
I can change if you wants prefer that! sure